site2/website/release-notes.md

```diff
@@ -1,6 +1,191 @@
+  * *Avro 1.9.1 enables the JSR310 datetimes by default, which might introduce some regression problems if users use generated source code by Avro compiler 1.8.x and contains datetimes fields. It's better to use Avro 1.9.x compiler to recompile.*
+* Support `unload` all partitions of a partitioned topic [#6187](https://github.com/apache/pulsar/pull/6187)  
+* Supports evenly distribute topics count when splits bundle [#6241](https://github.com/apache/pulsar/pull/6241)  
+* KeyValue schema support for pulsar sql [#6325](https://github.com/apache/pulsar/pull/6325)  
+* Bump netty version to 4.1.45.Final [#6424](https://github.com/apache/pulsar/pull/6424)  
+* Support BouncyCastle FIPS provider [#6588](https://github.com/apache/pulsar/pull/6588)  
+* Improve Key_Shared subscription message dispatching performance. [#6647](https://github.com/apache/pulsar/pull/6647)  
+* Add Joda time logical type conversion. [#6704](https://github.com/apache/pulsar/pull/6704)  
+
+
+#### Fixes
+
+* [Broker] Fixed unit test issue in BrokerServiceTest.java [#6006](https://github.com/apache/pulsar/pull/6006)  
+* [Broker] Fixed NPE occur when get partitioned topic stats [#6060](https://github.com/apache/pulsar/pull/6060)  
+* [Broker] Fixed zero queue consumer message redelivery [#6076](https://github.com/apache/pulsar/pull/6076)  
+* [Broker] Fixed message redelivery for zero queue consumer while using async api to receive messages [#6090](https://github.com/apache/pulsar/pull/6090)  
+* Fixed the argument and description for dead letter topic is wrong [#6101](https://github.com/apache/pulsar/pull/6101)  
+* [broker] Fixed bug that backlog message that has not yet expired could be deleted due to TTL [#6211](https://github.com/apache/pulsar/pull/6211)  
+* [Broker] Remove problematic semicolon from conf [#6303](https://github.com/apache/pulsar/pull/6303)  
+* [Zookeeper] Fixed casting in ZooKeeperCache.getDataIfPresent() [#6313](https://github.com/apache/pulsar/pull/6313)  
+* [Broker] Fixed broker to specify a list of bookie groups [#6349](https://github.com/apache/pulsar/pull/6349)  
+* Fixed Amend the default value of `supportedNamespaceBundleSplitAlgorithms` [#6374](https://github.com/apache/pulsar/pull/6374)  
+* Fix maven broken link [#6068](https://github.com/apache/pulsar/pull/6068)  
+* [Minor] Fixed java code errors reported by lgtm. [#6398](https://github.com/apache/pulsar/pull/6398)  
+* Consumer received duplicated deplayed messages upon restart [#6404](https://github.com/apache/pulsar/pull/6404)  
+* Fixed the max backoff configuration for lookups [#6444](https://github.com/apache/pulsar/pull/6444)  
+* Pin the netty-transport-native-epoll to avoid conflicts [#6194](https://github.com/apache/pulsar/pull/6194)  
+* Fix java doc for key shared policy. [#6341](https://github.com/apache/pulsar/pull/6341)  
+* Seek to the first one >= timestamp [#6393](https://github.com/apache/pulsar/pull/6393)  
```

Comment:
Looks can't get key information of this fix, could you please check the PR and refine it.